modules/oe_content_publication/oe_content_publication.info.yml

```diff
@@ -5,25 +5,40 @@ package: OpenEuropa
 type: module
 core: 8.x
 dependencies:
+  - composite_reference:composite_reference
```

Comment:
`field_group` is also new dependency that should be enabled.

Reply:
I altered the Title field label and added the `field_group` dependency.